debian/changelog

```diff
@@ -1,3 +1,9 @@
+nnstreamer (1.7.2.0) unstable xenial bionic; urgency=medium
```

Comment:
Shouldn't it be changed to focal instead of xenial now?

Reply:
Added focal.